### ISSUE TITLE
Oxide-msg receives an old/incorrect state

### ISSUE THREAD

**Issue by B1TC0R3** [DHBW-FN/OxideWM#162]
## Found in
    
- 5210cb5d218d4a5ed7dc1c674ecaa1f33a1547ad

## Description

When opening a window with the command
```bash
oxide-msg -c exec --args xterm
oxide-msg -c exex --args kitty
```
the state received by the command 
```bash
oxide-msg -c state
```
is one event behind.
After executing the command a second time, the current state is received, which validates the fact that there is a one-event delay.

## Comments

This bug is preventing the validation of automated test.

**Comment by FelixSchladt** [comment]
is it possible that the status bar failed to launch on the occasions where you got an incorrect state?

Because at my system it somehow works and I do not have issues with old states.

**Comment by FelixSchladt** [comment]
merged with #169
